Assignment/solution_student/index.html

```diff
@@ -4,30 +4,95 @@
     <link rel="stylesheet" href="css/style.css"><!-- Our site's stylesheet -->
 
     <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
   </head>
   <body>
+    
+    <header>
 
-    <!-- YOUR MARKUP GOES HERE! -->
-    <div class="container">
+      <h1>Relaxr</h1>
+<!--  -->
+        <nav>
+          <ul>
+            <li><a href="#" class="nav-links">About</a></li>
+            <li><a href="#" class="nav-links">FAQ</a></li>
+            <li><a href="#" class="nav-links">Team</a></li>
+            <li><a href="#" class="nav-links">Contact Us</a></li>
+            <li><a href="#" class="nav-links">Blog</a></li>
+          </ul>
+        </nav>
     
-      <header>
-        <h1>FEWD HTML Template</h1>
-      </header>
+    </header>
+
+    <main>
+
+    <div class="main clearfix">
+    <div class="main-content">
+
+      <h2> How I implemented Relaxr in 2 weeks and changed my life.</h2>
+
+        <img class= "main-images" src="images/blog_photo1.jpg">
+
+        <p class="first-character"> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. </p>
+
+        <p class="read-more">Read More ></p>
+
+      <hr>
+
+      <h2> I traveled across the globe thanks to Relaxr's Excel automation</h2>
+
+        <img class= "main-images" src="images/blog_photo2.jpg">
+
+        <p class="first-character">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+        <p class="read-more">Read More ></p>
+
+      <hr>
+
+    </main>
+
+      <div class="sidebar">
+
+      <h3>Categories</h3>
+
+        <ul>
+          <li class="sidebar-li">Success Stories</li>
+          <li class="sidebar-li">Stats</li>
+          <li class="sidebar-li">How tos</li>
+          <li class="sidebar-li">Best of</li>
+          <li class="sidebar-li">Business Trips</li>
+        </ul>  
+
+      <h3> About Relaxr</h3>
+
+          <p id="relaxr-text">Proin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla.</p>
+          <p id="learn-more">Learn More</p>
+
+      <img class="ad-unit" src="images/ad-unit.png">
+
+      </div>
+      </div>
```

Comment:
The first closing `div` here corresponds to the `<div class="sidebar">`, but the other two are stray closing tags, probably from tags you opened earlier in the doc and then rearranged, so you can remove them here. What I like to do to keep track of closing tags is add a comment after them to tell me the class or id of the opening tag it corresponds to, for example:
```
</div><!-- .sidebar -->
```